scripts/check/surface.py

```
@@ -0,0 +1,303 @@
+import subprocess
+import sys
+import shutil
+from dataclasses import dataclass
+from pathlib import Path
+from typing import cast
+
+SCRIPTS = Path(__file__).resolve().parents[1]
+if str(SCRIPTS) not in sys.path:
+    sys.path.insert(0, str(SCRIPTS))
+
+from lib.core import BaseCommandSettings, McbResult, get_logger  # ruff: ignore[module-import-not-at-top-of-file]
```

Ruff (0.16.2)

[warning] Invalid rule code in suppression: module-import-not-at-top-of-file

Enable `lint.preview` to use rule names

(RUF102)